content/Watsonx/WatsonxData/107.md

```diff
@@ -0,0 +1,76 @@
+---
+title: '107: Offloading Data from a Data Warehouse'
+timeToComplete: 45
+updated: 2023-09-20
+---
+
+## Offloading Data from a Data Warehouse
+
+
+With watsonx.data, enterprises can offload data that really doesn’t need to be in the warehouse to lower-cost object storage. With fit-for-purpose engines, data warehousing costs can be reduced by offloading workloads from a data warehouse to watsonx.data. Specifically, applications that need to access this data can query it through Presto (or Spark). This includes being able to combine the offloaded data with the data that remains in the warehouse. This section will show an example of how this can be done with Db2 (but the steps are equivalent for other data warehouse products).
+
+Additionally, external engines that support the Iceberg open table format can also work directly with data in watsonx.data’s object storage. Latest Db2 and Netezza can participate in the watsonx.data ecosystem as well, accessing the lakehouse data in object storage directly (just as Presto and Spark can).
+
+In the [previous Lab](/watsonx/watsonxdata/106) you registered an existing Db2 environment with watsonx.data (calling it **Db2DW**) and created a catalog for it called **db2catalog**.
+
+In this example scenario, you’re going to “move” the **gosalesdw.sls_sales_fact** table from Db2 to watsonx.data’s object storage. It will be created as an Iceberg table, in a new schema you create called **wxgosalesdw**, managed by the **iceberg_data** catalog.
+
+1. Select the Data manager icon from the left-side menu.
+
+2. Go to the top of the navigation pane and click the **Create** dropdown menu. Select **Create schema**. (Note: These steps are using the web interface, but you can also create a schema through SQL.)
+
+  ![](./images/107/create-schema.png)
+
+3. In the **Create schema** pop-up window, select/enter the following information, and then click the Create button:
+
+            • Catalog: iceberg_data
```

Comment:
Bullet points should be separated by newline characters so that they appear on separate lines.

Reply:
done